lib/galaxy/dependencies/dev-requirements.txt

```diff
@@ -2,16 +2,16 @@ aiohttp==3.9.1 ; python_version >= "3.8" and python_version < "3.12"
 aiosignal==1.3.1 ; python_version >= "3.8" and python_version < "3.12"
 alabaster==0.7.13 ; python_version >= "3.8" and python_version < "3.12"
 amqp==5.2.0 ; python_version >= "3.8" and python_version < "3.12"
-anyio==4.1.0 ; python_version >= "3.8" and python_version < "3.12"
```

Comment:
I'd only remove fastapi-utils without all the unrelated dependencies updates.

Reply:
Absolutely! I think I ran the update script, but I don't have to. I'll fix this.

Reply:
Done.